.changeset/grumpy-bags-whisper.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/itwinui-react": patch
+---
+
+Fixes bug with `InformationPanel` where changing orientation after resizing the panel would cause the panel to maintain it's resized width/height.
```

Comment:
Changesets should generally use past tense

"Fixes bug" → "Fixed a bug"

Reply:
Will do.

Reply:
Done in this [commit.](https://github.com/iTwin/iTwinUI/pull/1964/commits/2f362c6558331f5e67f75da885fa345549aafa2c)